API.md

```diff
@@ -32,6 +32,27 @@ This page documents the Survey service API endpoints. Apart from the Service Inf
 
 An `HTTP 204 No Content` status code is returned if there are no known surveys.
 
+## List Surveys by Survey Type
+* 'GET /surveys/surveytype/<type>'  Where type is one of Business,Social or Census
```

Comment:
I followed on from the style already there 
"/surveys/shortname/{shortName}"
"/surveys/{surveyId}/classifiertypeselectors/{classifierTypeSelectorId}"
"/surveys/ref/{ref}"
If I did change it then it would be the odd one out . Suggest that if we do this we do it for all endpoints as part of a wider ticket ?

Reply:
Yeah unfortunately the pattern is already wrong technically so I guess at least this is consistently wrong 😉 👍

Reply:
Raised a new issue #63 to cover this - probably needs cribbing to a card on the backlog too?

Reply:
also thought of this and decided not to upset the "consistently wrong" apple cart lol